src/python-lxc/lxc.c

```diff
@@ -77,6 +80,36 @@ static int lxc_wait_for_pid_status(pid_t pid)
     return status;
 }
 
+/* Copied from lxc/confile.c, with HAVE_SYS_PERSONALITY_H check removed */
+signed long lxc_config_parse_arch(const char *arch)
+{
+    struct per_name {
```

Comment:
I've removed HAVE_SYS_PERSONALITY_H check here, and just copied necessary constants from sys/personality.h. Not sure if it's the right way, but at least it doesn't require custom configure step, which is rather hard to implement correctly in distutils.